src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -637,6 +666,13 @@ def _db_dw_create(
     Handles common concerns such as setting location and sku properties.
     '''
 
+    # This check needs to be here, because server side logic of
+    # finding a default sku for Serverless is not yet implemented.
+    if kwargs['compute_model'] == ComputeModelType.serverless:
+        if sku is None or sku.tier is None or sku.family is None or sku.capacity is None:
+            raise CLIError('When creating a severless database, please pass in edition, '
```

Comment:
I have a small question here.  When  losting sku value, why do you let user pass -e -f and -c? How is it related to sku?

Reply:
This is a design issue from the server side code. If we want a Serverless compute model database, we actually do not have a default edition (-e), family (-f), and capacity (-c) configured for the customer. We are making a server side improvement for that, but at the mean time, letting customer provide the edition, family, and capacity as input params should be the most desired experience.